dts/arm/nuvoton/npcx/npcx7-miwus-int-map.dtsi

```diff
@@ -13,22 +13,22 @@
 
 			group_ad0: group-ad0-map {
 				irq        = <7>;
-				irq_prio   = <0>;
+				irq_prio   = <2>;
```

Comment:
At some point this should be irq-prio

Reply:
Oops. I  missed this node. I will browse all yaml, dts, and dtsi files again. A CL to fix this will be submitted later.

Reply:
Done.